### DISCUSSION TITLE
[@types/react-helmet] HelmetProvider lacks support for @types/react v18.x

### DISCUSSION THREAD

**Discussion by ashmortar** [DefinitelyTyped/DefinitelyTyped#59852] [Issues with a @types package] [answered]
@types/react has [removed implicit children](https://solverfox.dev/writing/no-implicit-children/) from `Component` and `FC`.  This breaks the `<HelmetProvider>` component as it does not specify children in its props
![Screen Shot 2022-04-12 at 12 16 19 PM](https://user-images.githubusercontent.com/33615526/163036930-f26eb859-a6e0-452f-9029-98e4c924140c.png)
.

[a comment by an automated account is not shown]

**Comment by wKovacs64** [comment, marked as answer]
@ashmortar `@types/react-helmet` doesn't have a `HelmetProvider` type - are you thinking of the [react-helmet-async](https://github.com/staylor/react-helmet-async) project by chance? If so, it looks like they ship their own types, so you'll probably want to raise an issue over there.

**Reply by ashmortar** [reply]
Ah, thank you and apologies
